front/app/components/admin/AddCollaboratorsModal/index.tsx

```diff
@@ -32,75 +33,83 @@ const AddCollaboratorsModal = ({
   const { formatMessage } = useIntl();
   const { data: appConfiguration } = useAppConfiguration();
   const { data: seats } = useSeats();
+  const [showSuccess, setShowSuccess] = useState(false);
 
   if (!appConfiguration || !seats) return null;
 
   const maximumCollaborators =
     appConfiguration.data.attributes.settings.core.maximum_moderators_number;
   const currentCollaboratorSeats =
     seats.data.attributes.project_moderators_number;
-  const hasReachedLimit =
+  const hasReachedOrIsOverLimit =
     !isNil(maximumCollaborators) &&
     currentCollaboratorSeats >= maximumCollaborators;
-  const buttonText = hasReachedLimit
+  const hasExceededSetSeats =
+    !isNil(maximumCollaborators) &&
+    currentCollaboratorSeats > maximumCollaborators;
+  const buttonText = hasReachedOrIsOverLimit
     ? formatMessage(messages.buyAdditionalSeats, {
         noOfSeats: noOfCollaboratorSeatsToAdd,
       })
     : formatMessage(messages.confirmButtonText);
 
+  const header = !showSuccess ? (
+    <Text color="primary" my="8px" fontSize="l" fontWeight="bold" px="2px">
+      {formatMessage(messages.giveCollaboratorRights)}
+    </Text>
+  ) : undefined;
```

Comment:
Minor nitpick: I use `null` instead of undefined if we explicitly set (aka define) a value.

Reply:
This is used as a prop. Best to use undefined as that has the same functionality as passing nothing. The added value there is also that in case there are default props, those would work well